scripts/deploy.sh

```diff
@@ -103,6 +104,24 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
   rm -f /etc/openstack_deploy/conf.d/ceilometer.yml
 fi
 
+# Apply host security hardening with openstack-ansible-security
+# The is applied as part of setup-hosts.yml
+if [[ "$DEPLOY_HARDENING" == "yes" ]]
```

Comment:
I think by this point we have ansible bootstrapped, yes? Why not simply use `lineinfile` to manage this section? I think it would be more readable, understandable, and maintainable that way.